octokit/search_test.go

```diff
@@ -0,0 +1,59 @@
+package octokit
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserSearchService_All(t *testing.T) {
```

Comment:
I think this can become `TestSearchService_UserSearch`, or just `TestUserSearch`.